packages/app/components/Today/OpenQueueCard.tsx

```diff
@@ -119,71 +127,55 @@ const OpenQueueCard = ({
 
   return (
     <PaddedCard
```

Comment:
Not sure what's causing it, but it looks like the header (that dark blue strip) at the top of each card no longer has rounded corners - it looks like our queue cards originally had rounded corners on all four sides
![image](https://user-images.githubusercontent.com/56804180/162234558-dbe3883a-df31-4204-b80f-cc749415f996.png)